tests/server/test_client.py

```diff
@@ -29,7 +32,7 @@ def httpx_mock_response(client, request: httpx.Request):
 
 
 @pytest.mark.parametrize("use_async", [False])
-def test_client_endpoints(httpx_mocked_response: HTTPXMock, use_async):
```

Comment:
Should there not be a type declaration here to satisfy mypy from your other PR?
If that is the case, removing the HTTPXMock type hint is not useful.

Reply:
No, mypy is perfectly happy to not have type-hints absolutely everywhere. This one is a bit awkward as its not clear what type is actually being returned by the fixture (and it might actually confuse mypy)